modules/olm-troubleshooting-operator-catalog-image.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * operators/olm-restricted-networks.adoc
+
+[id="olm-troubleshooting-operator-catalog-image_{context}"]
+= Troubleshooting
+
+[discrete]
+=== The catalog image build succeeded, but the final image does not contain any Operators.
+
+This can occur if the image repository specified by `--to` does not have Docker
+v2-2 enabled. As of November 2019, only select Quay namespaces have v2-2
+enabled.
+
+The OpenShift team keeps a
```

Comment:
s/OpenShift/{product-title}

Reply:
Not actually including this module in the assembly, so I'll just remove the module for now. Possibly re-include a troubleshooting section later if needed.